compiler/rustc_codegen_ssa/src/mir/rvalue.rs

```diff
@@ -502,6 +502,7 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             }
 
             mir::Rvalue::NullaryOp(mir::NullOp::SizeOf, ty) => {
+                let ty = self.monomorphize(&ty);
```

Comment:
do we need to do something similar in the cranelift backend?

Reply:
I am not familiar with cranelift backend, but it seems to take that into account already:

https://github.com/rust-lang/rust/blob/25f6938da459a57b43bdf16ed6bdad3225b2a3ce/compiler/rustc_codegen_cranelift/src/base.rs#L742-L751

Reply:
Perfect, thanks!